src/app/app-routing.module.ts

```diff
@@ -4,7 +4,10 @@ import { RouterModule, Routes } from '@angular/router';
 import { FilesortComponent } from './filesort/filesort.component';
 
 const routes: Routes = [
-  {path:'filesort',component:FilesortComponent}
+  {path:'filesort',component:FilesortComponent},
+  {path:'filesort/file',component:FilesortComponent},
```

Comment:
why for different paths same components?

Reply:
This path help me understand what service is actual and what api I must work